### PR TITLE
adding more resources: faq and HOC

### PR DESCRIPTION
**Pull request by da-vaibhav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -444,6 +445,7 @@ A collection of awesome things regarding React ecosystem.
 * [react-pagespeed-score - A React component for display a dial-type chart of PageSpeed Insights](https://github.com/cauealves/react-pagespeed-score)
 * [react-livestamp - A simple and html date countdown component for React](https://github.com/markafoni/react-livestamp)
 * [react-offcanvas - Off-canvas menus for React](https://github.com/vutran/react-offcanvas)
+* Higher-order Components (https://gist.github.com/sebmarkbage/ef0bf1f338a7182b6775)
```
Comment (paulmelnikow) [inline README.md]: This link is okay but doesn't really stand alone. We should link to an article.

File: README.md [hunk 2]
```diff
@@ -106,6 +106,7 @@ A collection of awesome things regarding React ecosystem.
 * [React Official Site](http://facebook.github.io/react/)
 * [React GitHub](https://github.com/facebook/react)
 * [React Community GitHub](https://github.com/reactjs)
+* [React FAQ] (https://github.com/timarney/react-faq)
```
Comment (paulmelnikow) [inline README.md]: This should link to the site, not the github page.

**Comment by da-vaibhav** [comment]
update the site link and removed the gist link.

File: README.md [hunk 3]
```diff
@@ -106,6 +106,7 @@ A collection of awesome things regarding React ecosystem.
 * [React Official Site](http://facebook.github.io/react/)
 * [React GitHub](https://github.com/facebook/react)
 * [React Community GitHub](https://github.com/reactjs)
+* [React FAQ] (https://reactfaq.site/)
```
Comment (Haroenv) [inline README.md]: there's an extra space here 😄

**Pull request merged**